src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
+});
+
+export default store;
```

Comment:
- [ ] @Chandan-devs-tech Great work adding your store however you forgot to import the books and categories reducers in your store.